hook/index.rb

```diff
@@ -14,6 +14,7 @@
   me = Docker::Container.me
   containers = Docker::Container.all - [me]
 end
+containers.sort_by! { |c| c.name }
```

Comment:
Thanks!!!

![](http://themodernape.com/wp-content/uploads/2014/09/internet-troll.jpg)

Yeah actually contributed without run on my machine 🙈 , confused how to run yaichi with a github fork, so bad! 🙇